slave/buildslave/runprocess.py

```diff
@@ -317,7 +320,7 @@ def to_str(cmd):
         if not os.path.exists(workdir):
             os.makedirs(workdir)
         if environ:
-            for key, v in environ.iteritems():
+            for key, v in iteritems(environ):
```

Comment:
I was mistaken. We **do** have to use 
```
iteritems(environ)
```



dict.iteritems()  does not work in python3